Sources/OpenTelemetrySdk/Trace/TracerSdkProvider.swift

```diff
@@ -78,4 +78,9 @@ public class TracerSdkProvider: TracerProvider {
         }
         sharedState.stop()
     }
+
+    ////Requests the active span processor to process all span events that have not yet been processed.
```

Comment:
is there a difference in swift - how many slashes comment starts wih? In this PR - there are 2, 3 and 4 slashes comments

Reply:
Yes, when using 3 slashes Xcode parses the code to generate documentation, two slashes used for standard comments . This 4 slashes was a typo :)